bakery/pipeline.yml

```diff
@@ -1,9 +1,9 @@
 ---
 resource_types:
-  - name: slack
+  - name: workflow-events
     type: docker-image
     source:
-      repository: karenc/concourse-slack-resource
+      repository: openstax/workflow-event-resource
```

Comment:
We have the updated resource now, to replace the one here. 
https://hub.docker.com/r/openstax/output-producer-resource

Reply:
I'm going to update both pipelines to use output-producer-resource after this.